.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,51 @@
+---
+name: 🐞 Bug Report
+about: Something went awry and you'd like to tell us about it.
+
+---
+
+<!--
+  ⚡️ katchow! We 💛 issues.
+
+  Please - do not - remove this template.
+  Please - do not - skip or remove parts of this template.
```

Comment:
If we leave this line here, then it should be made clear which parts *can* be skipped if they do not apply. See below.